include/nds/arm9/video.h

```diff
@@ -37,10 +37,10 @@
 	uses. They can hold the graphics for your sprites, the textures for your 3D
 	space ships, the tiles for your 2D platformer, or a direct map of pixels to
 	render to the screen. Figuring out how to effectively utilize this flexible but
-	limited amount of memory will be one the most challenging endeavors you will
```

Comment:
Both of these are a correct spelling, but I think we tend to stick with the common forms found in U.S. English, in this case "endeavor".